src/sentry/static/sentry/app/components/eventOrGroupHeader.tsx

```diff
@@ -26,7 +26,7 @@ type Props = WithRouterProps<{orgId: string}> & {
   hideLevel?: boolean;
   query?: string;
   className?: string;
-} & DefaultProps;
+} & Partial<DefaultProps>;
```

Comment:
Why do default props need to be partialled? Is typescript not able to figure that out?

Reply:
I am having this issue because of the `withRouter`